eyelid-match-ops/src/primitives/yashe/test/encdec.rs

```diff
@@ -28,5 +28,5 @@ fn encrypt_decrypt_test() {
     //
     // TODO: find a config that does work and use it for TestRes/TinyPoly.
     encrypt_decrypt_helper::<FullRes>();
-    encrypt_decrypt_helper::<IrisBits>();
+    //encrypt_decrypt_helper::<IrisBits>();
```

Comment:
Let's keep test coverage with multiple configs, either `IrisBits` or `Tiny...` would be ok.

Otherwise we might hard-code assumptions about the `FullRes` config in production code.